### ISSUE TITLE
SAR test aug

### ISSUE THREAD

**Issue by andyjpaddle** [open-mmlab/mmocr#429]
hi, I have test images of the public OCR dataset with the pre-trained SAR model . I compared the different test strategies(test with aug or test no aug) on four datasets, IC13, IC15, IIIT5K, SVT. 

According to the eval results, I find only IC15 have different results
   aug   vs   no aug
(78.86 vs 76.79)

the rest of the datasets are  almost same:
            aug     vs  no aug
IIIT5K: 95.17  vs  95.23
SVT: 89.95    vs  89.95
IC13: 93.79    vs  93.89

So does the test aug strategy works depend on the datasets?

**Comment by cuhk-hbsun** [comment]
Yes, it indeed depends on the datasets. `test_time_aug` has an effect only on the image which height is larger than width, while most of the images are not. 
For images whose width are larger than height (majority), `test_time_aug` has no effect. Only original image will be used for inference.
For images whose height are larger than width (minority), the image triplet (original image, image rotated 90 degree clockwise, image rotated 90 degree anti-clockwise) will be used for infernece and the result with the hightest score will be selected.

**Comment by paulkuo123** [comment]
@andyjpaddle Could you upload your training config? I trained a SAR model but got a bad accuracy on these four datasets.
